### PR TITLE
Flowsheet Visualizer: Multi connections for the mixer component as a 'target' model

### PR DESCRIPTION
**Pull request by elbashandy**
## Fixes

- Supporting multi-connection links for the 'Mixer' component as a destination/target end.
- Refactored the way we load ports that links connect to in order to support the multi-connection feature.

## Summary/Motivation:
Providing an initial way for developers to control the orthogonal path that links should go through when connecting components. This provides a flexible way for developers to change the ways a Unit Model  (e.g. Mixer) should be connected to by links/edges. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by dangunter** [comment]
@elbashandy I am seeing test failures in CI. Are you seeing those locally?

**Comment by dangunter** [comment]
@elbashandy I tried a flowsheet with the mixer unit and it doesn't show any differently

File: idaes/ui/fsvis/static/js/cell_config.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
```
Comment (dangunter) [inline idaes/ui/fsvis/static/js/cell_config.js:1]: Could you add a file header that describes what this file does

File: idaes/ui/fsvis/static/js/main.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Paper } from './paper.js';
```
Comment (dangunter) [inline idaes/ui/fsvis/static/js/main.js:1]: Could you add a file header that says what this file is for

File: idaes/ui/icons.py [hunk 3]
```diff
@@ -140,97 +158,549 @@ def _build_link_positions(self) -> Dict:
     #  - Use 'cstr' as your template for new entries
     #  - Do not remove in/out entries in existing entries, or arcs won't connect
     _mapping = {
```
Comment (dangunter) [inline idaes/ui/icons.py:160]: Could you add a TODO for moving this mapping to its own directory/files

**Review by dangunter — CHANGES_REQUESTED** [review]
Minor documentation changes requested

**Review by dangunter — APPROVED** [review]
LGTM

**Review by ksbeattie — APPROVED** [review]
LGTM

**Pull request merged**